.envrc

```diff
@@ -14,6 +15,29 @@ export LNDONCHAIN_MACAROON=""
 export MACAROONOUTSIDE1=$(cat dev/lnd/$NETWORK/lnd-outside-1.admin.macaroon.base64)
 export MACAROONOUTSIDE2=$(cat dev/lnd/$NETWORK/lnd-outside-2.admin.macaroon.base64)
 
+fetch_loop_macaroon() {
```

Comment:
could those macaroon and tls be injected like @openoms has done for lnd?

Reply:
yep! done! https://github.com/GaloyMoney/galoy/pull/1396/commits/fabb350da08f693783af28701df02f526aba8e21